launcher/container_runner_test.go

```diff
@@ -540,3 +545,106 @@ func TestInitImageDockerPublic(t *testing.T) {
 		}
 	}
 }
+
+func TestMeasureCELEvents(t *testing.T) {
+	ctx := context.Background()
+	fakeContainer := &fakeContainer{
+		image: &fakeImage{
+			name:   "fake image name",
+			digest: "fake digest",
+			id:     "fake id",
+		},
+		args: []string{"fake", "args"},
+		env:  []string{"fake", "env"},
+	}
+
+	testCases := []struct {
+		name          string
+		wantCELEvents []cel.CosType
+		launchSpec    spec.LaunchSpec
+	}{
+		{
+			name:          "measure container events and launch separator event",
+			wantCELEvents: []cel.CosType{cel.LaunchSeparatorType, cel.ImageIDType, cel.ImageDigestType},
+			launchSpec:    spec.LaunchSpec{},
+		},
+		{
+			name:          "measure memory monitoring event and launch separator event",
+			wantCELEvents: []cel.CosType{cel.LaunchSeparatorType, cel.MemoryMonitorType},
```

Comment:
`wantCELEvents` should have the separator last.

Reply:
and above

Reply:
Done.